agent-dispatchers/logger/src/main/resources/logback.xml

```diff
@@ -0,0 +1,15 @@
+<configuration>
```

Comment:
Does packaging logback.xml in this module a good idea? How does it work when multiple logback.xmls exist on the classpath.
Should we put the configuration just for this logger in top level logback.xml?

Reply:
Good question, I think we should not do that as we might override the logging configuration for those consuming this jar. I will remove it.

Reply:
Good question, my intention was that we give special formatting to this logger as this is a very specific use case. In one hand I would be OK dropping this as the format might not be that important given that this is not for production purposes, in the other hand if we can find a way to use this `logback.xml` for this module that would be great if it does not involve too much engineering. Ping @anuraaga as my java reference.

Reply:
Yeah I've never seen multiple patterns for different loggers in practice - I think managing the logs tends to become difficult for the end-user, who may need to parse a specific format (especially if using JSON). So recommend not having this here, and have it as `logback-test.yml` in the test resources so test log output is displayed.